src/main/java/ch/njol/skript/lang/parser/ParserInstance.java

```diff
@@ -532,8 +533,10 @@ public void deleteCurrentSkriptEvent() {
 	public void setCurrentScript(@Nullable Config currentScript) {
 		if (currentScript == null)
 			return;
-		//noinspection ConstantConditions - shouldn't be null
-		Script script = ScriptLoader.getScript(currentScript.getFile());
+		File file = currentScript.getFile();
+		if (file == null)
+			return;
+		Script script = ScriptLoader.getScript(file);
```

Comment:
Unlikely, but it's possible a user could somehow feed a missing/unlinked/alias/folder in as the file object. Might be worth checking it *is* as file as well as not-null.

Reply:
I think this would just result in a failure to set the current script since getScript would likely return null